packages/device_info/device_info/lib/device_info.dart

```diff
@@ -3,28 +3,26 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'package:device_info_platform_interface/device_info_platform_interface.dart';
 
-import 'package:flutter/services.dart';
+export 'package:device_info_platform_interface/model/android_device_info.dart';
+export 'package:device_info_platform_interface/model/ios_device_info.dart';
```

Comment:
Ah, forgot about the current syntax for this, so you don't have to access "private" parts of the platform_interface package. Do it like this:

`export 'package:device_info_platform_interface/device_info_platform_interface.dart' show ClassName1, ClassName2, ClassName3;`

Apologies, I should have been clearer. You can see this in other plugins, like [this extreme case in google_maps_flutter](https://github.com/flutter/plugins/blob/82f8f490cfa4b257fc987cd40bab983a8c1ab2e6/packages/google_maps_flutter/google_maps_flutter/lib/google_maps_flutter.dart#L20-L46).

Please, make sure you're only re-exporting the classes that you moved to the platform_interface.

The problem with the current version is that if we add by accident some code to those files, it gets reexported automatically, whether we want it or not.

Reply:
No need for an apology sir, I am learning many things from your reviews and it's a great learning this way. BTW I have pushed the changes.